### PR TITLE
feat(content): add `no-scroll` attribute to disable scrolling

### LINKED ISSUE #7644: How to disable scroll for ion-content in ionic 2?
**Note: If you are having problems formatting your issue please refer to this article on using markdown in Github:**

https://guides.github.com/features/mastering-markdown/

**Note: for support questions, please use one of these channels:** 

https://forum.ionicframework.com/
http://ionicworldwide.herokuapp.com/
#### What behavior are you expecting?

**Steps to reproduce:**
1. 
2. 
3. 

```
insert any relevant code between the above and below backticks
```

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

**Which Ionic Version?** 1.x or 2.x
#### Plunker that shows an example of your issue

**For Ionic 1 issues** - http://plnkr.co/edit/Xo1QyAUx35ny1Xf9ODHx?p=preview

**For Ionic 2 issues** - http://plnkr.co/edit/me3Uk0GKWVRhZWU0usad?p=preview

**Run `ionic info` from terminal/cmd prompt:** (paste output below)

### PR DESCRIPTION
**Pull request by Manduro**
#### Short description of what this resolves:
Allow to disable scrolling on `ion-content`.

#### Changes proposed in this pull request:

- Add `no-scroll` attribute to disable scrolling
- Document `ion-fixed` and `no-bounce` attributes

**Ionic Version**: 3.x

**Fixes**:

Closes #7644
Closes ionic-team/ionic-site#940
References #8422 #10329 #10734

### CODE REVIEW

File: src/components/content/content.scss [hunk 1]
```diff
@@ -53,6 +53,13 @@ ion-content.js-scroll > .scroll-content {
   will-change: initial;
 }
 
+ion-content[no-scroll] > .scroll-content {
+  overflow-x: initial;
+  overflow-y: initial;
+  -webkit-overflow-scrolling: auto;
+  will-change: initial;
```
Comment (AmitMY) [inline src/components/content/content.scss]: Instead of initial, shouldn't it be `overflow: hidden;`?

Comment (Manduro) [inline src/components/content/content.scss]: @AmitMY I used this as `ion-content.js-scroll > .scroll-content` uses it as well, so I figured there might be a reason. Wouldn't matter much anyway as its contained elsewhere as well. Will change it for both if requested!

Comment (AmitMY) [inline src/components/content/content.scss]: If it is somewhere else, that is good. You can perhaps then unify both to:
```
ion-content.js-scroll > .scroll-content {
  position: relative;

  min-height: 100%;
}

ion-content.js-scroll > .scroll-content,
ion-content[no-scroll] > .scroll-content {
  overflow-x: initial;
  overflow-y: initial;
  -webkit-overflow-scrolling: auto;
  will-change: initial;
}
```

to remove the code clone, only if you think that is ok.

**Comment by Ionitron** [comment]
Hello and thank you for contributing to Ionic! We have been working on porting all of the Ionic components to web components and have recently updated `master` to reflect this. This significant change has caused this pull request to break. While we really appreciate the time and effort you put into creating this, we are not able to merge it because of the newly introduced conflicts. We are extremely sorry about this. We will not be merging any more features in to v3. If this is a feature and you have the time, please resubmit this PR against the `master` branch. If this is a critical security issue in v3, we would greatly appreciate it if you would resubmit the PR against the new `v3` branch. Thanks so much for your time!

**Pull request closed without merging**